.github/workflows/deploy_branch_preview.yml

```diff
@@ -60,8 +65,21 @@ jobs:
             ${{ runner.os }}-
       - name: Install Packages
         run: npm install
+      - name: Get Latest Release
+        id: latestRelease
+        run: |
+          RELEASE_TAG=$(echo $(git describe --tags --abbrev=0))
+          echo "::set-output name=release::$RELEASE_TAG"
```

Comment:
Since we output this directly, won't the `release` be `prod-x.x.x`, when we actually want `x.x.x`?

Reply:
hmmm true